src/NuGet.Clients/NuGet.VisualStudio.Implementation/VsTemplateWizard.cs

```diff
@@ -100,15 +100,15 @@ private IEnumerable<PreinstalledPackageConfiguration> GetConfigurationsFromVsTem
                 var isPreunzippedString = packagesElement.GetOptionalAttributeValue(IsPreunzippedAttributeName);
                 if (!string.IsNullOrEmpty(isPreunzippedString))
                 {
-                    Boolean.TryParse(isPreunzippedString, out isPreunzipped);
+                    bool.TryParse(isPreunzippedString, out isPreunzipped);
```

Comment:
I don't see anything related to async in this file. Consider reverting.

Reply:
Yeah, I had it opened for some reason, and those are red Intellisense errors, so I fixed them to go away. 

Reverting, in any case.